contracts/traits/psp34/psp34.rs

```diff
@@ -1,25 +1,66 @@
 pub use crate::traits::errors::{
-    PSP721Error,
-    PSP721ReceiverError,
+    PSP34Error,
+    PSP34ReceiverError,
 };
+use ink_storage::traits::{
+    PackedLayout,
+    SpreadLayout,
+};
+#[cfg(feature = "std")]
+use ink_storage::traits::StorageLayout;
 use brush::traits::AccountId;
-use ink_prelude::vec::Vec;
+use ink_prelude::{
+    string::String,
+    format,
+    vec::Vec,
+    vec,
+};
+use brush::traits::Balance;
+
+// Id is an Enum and its variant are types
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, scale::Encode, scale::Decode, SpreadLayout, PackedLayout)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo, StorageLayout))]
+pub enum Id {
+    U8(u8),
+    U16(u16),
+    U32(u32),
+    U64(u64),
+    U128(u128),
+    Bytes(Vec<u8>),
+}
 
-pub type Id = [u8; 32];
+impl From<Id> for Vec<u8> {
```

Comment:
You don't need that=)

Reply:
After the change above, yes, we don't need it